content/en/containers/kubernetes/tag.md

```diff
@@ -103,8 +103,350 @@ com.datadoghq.ad.tags: '["<TAG_KEY>:TAG_VALUE", "<TAG_KEY_1>:<TAG_VALUE_1>"]'
+
+
+```yaml
+datadog:
+  kubernetesResourcesLabelsAsTags:
+    nodes:
+      <NODE_LABEL>: <NODE_TAG_KEY>
+    deployments.apps:
+      <DEPLOYMENT_LABEL>: <DEPLOYMENT_TAG_KEY>
+```
+
+For example, you could set up:
+
+```yaml
+datadog:
+  kubernetesResourcesLabelsAsTags:
+    nodes:
+      kubernetes.io/arch: arch
+    deployments.apps:
+      foo: bar
+    pods:
+      baz: qux
+```
+
+For Agent v7.24.0+, use the following environment variable configuration to add all resource labels as tags to your metrics. In this example, the tags' names for statefulsets are prefixed by `<PREFIX>_`:
+
+```yaml
+datadog:
+  kubernetesResourcesLabelsAsTags:
+    statefulsets.apps:
+      "*": <PREFIX>_%%label%%
+```
+{{% /tab %}}
+
+{{% tab "Manual (DaemonSet)" %}}
+
+Each resource type should be specified in the format `resourceType.apiGroup`, where `resourceType` is the plural name of the resource.
+
+If a specific resource is in the empty api group (for example `pods` and `nodes`), it can be specified using `resourceType`.
+
+To extract a given node label `<NODE_LABEL>` and a given deployment label `<DEPLOYMENT_LABEL>` and transform them as tag keys `<NODE_TAG_KEY>` and `<DEPLOYMENT_TAG_KEY>` within Datadog, add the following environment variable to the Datadog Agent:
+
+```bash
+ DD_KUBERNETES_RESOURCES_LABELS_AS_TAGS='{"nodes":{"<NODE_LABEL>": "<NODE_TAG_KEY>"},"deployments.apps":{"<DEPLOYMENT_LABEL>": "<DEPLOYMENT_TAG_KEY>"}}'
+```
+
+For example, you could set up:
+
+```bash
+DD_KUBERNETES_RESOURCES_LABELS_AS_TAGS='{"nodes":{"kubernetes.io/arch": "arch"},"deployments.apps":{"foo":"bar"},"pods":{"baz":"qux"}}'
+```
+
+For Agent v7.24.0+, use the following environment variable configuration to add all resource labels as tags to your metrics. In this example, the tags' names for statefulsets are prefixed by `<PREFIX>_`:
+
+```bash
+DD_KUBERNETES_RESOURCES_LABELS_AS_TAGS='{"statefulsets.apps":{"*": "<PREFIX>_%%label%%"}}'
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+**Notes**: Custom metrics may impact billing. See the [custom metrics billing page][3] for more information.
+
+<div class="alert alert-info">
+
+This configuration option is merged with other configurations set in <a href="/containers/kubernetes/tag/#pod-labels-as-tags">podLabelsAsTags</a>, <a href="/containers/kubernetes/tag/#namespace-labels-as-tags">namespaceLabelsAsTags</a> and <a href="/containers/kubernetes/tag/#node-labels-as-tags">nodeLabelsAsTags</a>. In case of conflict, <a href="/containers/kubernetes/tag/#kubernetes-resources-labels-as-tags">`kubernetesResourcesLabelsAsTags`</a> take precedence while merging the configurations.
+
+For example, if you have the following configurations:
+
+```yaml
+datadog:
+  kubernetesResourcesLabelsAsTags:
+    pods:
+      foo: bar
+      baz: qux
+
+  podLabelsAsTags:
+    foo: quux
+    bar: quuz
+```
+
+The following mapping will be used to extract tags from pod labels:
+
+```yaml
+foo: bar
+baz: qux
+bar: quuz
+```
+
+</div>
+
+{{< /tabs >}}
```

Comment:
This is either an extra unnecessary closing tag, or you are missing an opening one. It is causing the check to fail.

Reply:
Indeed, this is an extra unnecessary closing tag.

I removed it, but the pipeline still fails.

Reply:
Let me try now.